### PR TITLE
fix(spellcheck): allow words with numbers in dictionary

### PR DESCRIPTION
**Pull request by petrjasek**
and spellcheck such words

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: client/app/scripts/superdesk/editor/spellcheck/spellcheck.js [hunk 1]
```diff
@@ -123,16 +121,21 @@ function SpellcheckService($q, api, dictionaries, editor) {
...
+
         return getDict().then(function() {
-            var words = text.match(/[a-zA-Z\u00C0-\u1FFF\u2C00-\uD7FF]+/g),
+            var words = text.match(/[0-9a-zA-Z\u00C0-\u1FFF\u2C00-\uD7FF]+/g),
                 errors = [];
             angular.forEach(words, function(word) {
-                var lowerWord = word.toLowerCase();
-                if (!dict[lowerWord]) {
-                    errors.push(word);
+                if (isNaN(word)) {
```
Comment (plamut) [inline client/app/scripts/superdesk/editor/spellcheck/spellcheck.js]: Just checking - do we indeed want to interpret strings like `1e3` and `0x5` as numbers and not report them as errors?

**Comment by amagdas** [comment]
:+1:

File: client/app/scripts/superdesk/editor/spellcheck/spellcheck.js (hunk 1 above)
Comment (petrjasek) [inline client/app/scripts/superdesk/editor/spellcheck/spellcheck.js]: yep exactly

**Pull request merged**